eos_data_distribution/names.py

```diff
@@ -19,5 +19,5 @@
 
 from pyndn import Name
 
-SUBSCRIPTIONS_INSTALLED = Name('/com.endlessm/subscriptions/installed')
-SUBSCRIPTIONS_SOMA = Name('/com.endlessm/subscriptions/soma')
+SUBSCRIPTIONS_INSTALLED = Name('/endlessm/subscriptions/installed')
```

Comment:
I think it would be worthwhile to put some of the explanation from your commit message into a comment here, so it’s easy to find.